packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -142,16 +142,66 @@ export function intervalsToOptions({ intervals = defaultIntervals }: { intervals
   SelectableValue<string>
 > {
   const options: Array<SelectableValue<string>> = intervals.map((interval) => {
-    const duration = parseDuration(interval);
+    const duration = parseDuration(getIntervalTranslated(interval));
     const ariaLabel = formatDuration(duration);
 
     return {
-      label: interval,
-      value: interval,
+      label: getIntervalTranslated(interval),
+      value: getIntervalTranslated(interval),
       ariaLabel: ariaLabel,
     };
   });
 
   options.unshift(translateOption(RefreshPicker.offOption.value));
   return options;
 }
+
+export function getIntervalTranslated(interval: string): string {
```

Comment:
I think a better way to translate this would be to just translate the letter `s` and then whenever we use an interval that is seconds it could translate that correctly. Because the intervals are configurable in the dashboard settings, so if someone adds `15s` then all the other ones would be translated except for that one. Maybe we could add a helper inside `durationutil`, however this would mean adding translations inside `grafana-data` which we don't at the moment.